### ISSUE TITLE
Djanghor's Year Beast primary attribute is always Spirit, but ability tooltip says it varies according to Q state

### ISSUE THREAD

**Issue by DeckardCainRPC** [ChalkyBrush/roshpit-bug-tracker#1460]
i actually think the tooltip should be amended to state that the Year Beast's primary attribute is Spirit, it's neat to play with this new attribute

**Comment by redledcat** [comment]
isn't it R state? spirit for monkey, etc?

**Comment by DeckardCainRPC** [comment]
yes, for his other three forms, but the Year Beast is strictly always a Spirit hero. this conflicts with his ability tooltip, which says that his primary attribute depends on the active "Mark" (Q buff).

**Comment by redledcat** [comment]
My bad, misread
